salesagility/SuiteCRM
# Fixed - System email template access issue

## Description



This change adds a new type "System" to email templates and assigns this type to system generated emails. This type cannot be edited by anyone other than an admin user.

## How To Test This

1. Check that system created email templates are set to type "system" on install.
2. Try to edit an email template with type "System" as non-admin.

## Types of changes


### Final checklist
